src/rabbit_amqqueue.erl

```diff
@@ -1082,6 +1245,8 @@ cancel_sync_mirrors(#amqqueue{pid = QPid}) ->
 cancel_sync_mirrors(QPid) ->
     delegate:invoke(QPid, {gen_server2, call, [cancel_sync_mirrors, infinity]}).
 
+is_mirrored(#amqqueue{type = quorum}) ->
```

Comment:
What do you mean? They're currently considered mirrored.

Reply:
I think the question is whether a quorum queue with a single member should be considered mirrored? I'm not sure what effect it would have to return false in this instance? what does it mean to be mirrored? Does it imply fault tolerance somehow? what are it's availability expectations? would a quorum queue with 2 members (NB: can handle no failures) be considered mirrored?

Reply:
The question is: what does is_mirrored mean?
Do we know all the places where this function can possibly be called and expectations during those calls?
What if the code which calls this function will assume that there are components of the mirrored queue like GM, master and slaves?
What properties should the person, who writes the code expect from the queue, which returns `true` here? Do we have an API (behaviour) for that?
When we had one type of mirroring it was two behaviours possible (is_mirrored: true or false)
Now we have three. Should this function return three possible results?

Reply:
In this case, the 'is_mirrored' definition means that there could be another incarnation of the queue in a different node (may it be HA mirror or quorum follower). Despite being an exported function, it's only used inside `rabbit_amqqueue` to determine which queues to delete on node down and when to retry `rabbit_amqqueue:with` operations.